skin/src/vue/c-content.vue

```diff
@@ -213,6 +213,45 @@ onUpdated(() => {
   justify-content: center;
 }
 
+/*TABLE STYLES*/
+/*TODO: Check a better way to write these styles without the global tag. Currently impossible to use :deep because the html inside the document content is not assigned an ID */
+
+:global(.document-content table) {
+  max-width: 100%;
+  width: 100%;
+  border: none;
+  border-collapse: collapse;
+  color: inherit;
+  margin-bottom: var(--cr-spacing-small);
+}
+
+:global(.document-content table tr) {
+  border-bottom: solid 1px var(--cr-input-border-color);
+}
+
+:global(.document-content table th) {
+  font-weight: var(--cr-font-weight-bold);
+  text-align: left;
```

Comment:
```suggestion
  text-align: start;
```

Cristal shouldn't repeat the mistake of XWiki to not write styles with RTL languages in mind, in particular as there are easy ways to keep the CSS generic like using `start` instead of `left` for `text-align`.

Reply:
Agreed, thanks for the heads up. I will make a pass on the source for other instances where this can be applied.